frontend/components/common/ProfilePic.js

```diff
@@ -0,0 +1,60 @@
+import s from 'styled-components'
+import { PROPIC_BACKGROUND, PROPIC_TEXT } from '../../constants/colors'
+
+const hashCode = s => {
+  let h = 0
+  let i = 0
+  if (s.length > 0) {
+    while (i < 1) {
+      h = (h << 5) - h + s.charCodeAt(i++) | 0
+    }
+  }
+  return h
+}
+
+const Placeholder = s.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: ${({ fontSize }) => fontSize || '1.5em'};
+  ${({ isRound }) => isRound ? 'border-radius: 9999px;' : ''}
+
+  ${({ backgroundColor }) => backgroundColor
+    ? `background-color: ${backgroundColor};`
+    : ''}
+  ${({ textColor }) => textColor
+    ? `color: ${textColor};`
+    : ''}
```

Comment:
Is there ever a case where these are not defined? If so, then we should have a default color. If not, then there's no need for the ternary operator

Reply:
I'll pull the first color constants from both color constant arrays and use that as the default.